server/db.go

```diff
@@ -669,6 +718,21 @@ func (c *Server) rotateIssuersV3() error {
 	return nil
 }
 
+// deleteIssuerKeys deletes issuers keys that have ended more than the duration ago.
+func (c *Server) deleteIssuerKeys(duration string) (int64, error) {
```

Comment:
is this safe for non-v3 rotated keys / use cases?

Reply:
cc @Sneagan

Reply:
https://github.com/brave-intl/challenge-bypass-server/pull/252/commits/111fb321527d10c2e11b29e05e8890bfbb56565b

fixed to apply only to v3 issuer keys.